src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -228,6 +230,21 @@ public void handleHelp() {
         }
     }
 
+    /**
+     * Opens the stats window or focuses on it if it's already opened.
+     */
+    @FXML
+    private void handleStats(CommandResult commandResult) {
+        this.statsWindow = new StatsWindow(commandResult.getNames(),
+                commandResult.getPercentages(), commandResult.getTitle());
+        if (!statsWindow.isShowing()) {
+            statsWindow.show();
+        } else {
+            helpWindow.focus();
```

Comment:
possible bug:
wrong window?
```suggestion
            statsWindow.focus();
```

Reply:
potential bug? should resolve this. after which i think its good to merge